faceted_search/package.json

```diff
@@ -7,8 +7,13 @@
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
```

Comment:
Are all these still needed with the latest changes, or were some of them used in code that has been removed?